book/index.md

```diff
@@ -39,6 +39,55 @@ API resources in the second row (PipelineRun, Snapshot) should primarilly be tho
 data-plane resources. The system responds to user requests by creating and managing the lifecycle of
 these resources.
 
+- An [Application] represents a functionally coherent set of git branches that should be built,
+  tested, and released together. The user provides and names their [Applications]. They are
+  generally long-lived and don't change much after they are created.
+- A [Component] represents an image to build from a particular git branch (and a particular context
```

Comment:
Should we use "image" or some other term? OCI artifact? Can we go ahead and generalize it _now_ or should we live with this definition and update later?

(this would need to be consistent throughout the proposed change)

Reply:
Oh, why not. Let's say OCI artifact.